great_expectations/expectations/metrics/map_metric_provider/map_condition_auxilliary_methods.py

```diff
@@ -176,7 +176,7 @@ def _pandas_map_condition_query(
         boolean_mapped_unexpected_values,
         compute_domain_kwargs,
         accessor_domain_kwargs,
-    ) = metrics.get("unexpected_condition")
+    ) = metrics["unexpected_condition"]
```

Comment:
Are we sure this will always be there? If this was `None` would is break the code below? If so I think this change is good.

Reply:
Since we're unpacking this currently, we'd get `TypeError: cannot unpack non-iterable NoneType object` if that were ever null. My change would move it to a `KeyError`. I think we're safe.